public/js/utils/source-map-worker.js

```diff
@@ -22,21 +33,21 @@ function enableSourceMaps() {
   sourceMapsEnabled = true;
 }
 
-function _resolveSourceMapURL(source) {
-  if (path.isURL(source.sourceMapURL) || !source.url) {
+function _resolveSourceMapURL(source: Source) : string {
+  const { url, sourceMapURL } = source;
```

Comment:
Please don't destructure this, the code is clearer by simply referencing `source` below. Unless you did this for a specific flow reason?